.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -727,7 +727,7 @@ test-upload-flow:
     - section_end "Checks Whether to Trigger a Test Upload"
 
     - section_start "Create Testing Branch"
-    - export BRANCH="${CI_COMMIT_BRANCH}-upload_test_branch-$(date +%s)"
+    - export BRANCH="${CI_COMMIT_BRANCH}-upload_test_branch-${CI_PIPELINE_ID}"
```

Comment:
this will keep the branch name unique ,but will still allow to easily find the generated branch name when stopping running pipelines